linux-btrfs-nas-server.html

```diff
@@ -148,9 +148,10 @@ <h3 style="color:#EB6841; font-size:28px; text-align:center;">NAS that just work
           </p>
           <h4>Solution:</h4>
           <p class="lead">
-            Rockstor is the first Linux, BTRFS based NAS distribution to help users build and maintain a NAS server with least effort. It's based on
-            CentOS 7, but comes with a later kernel from Elrepo and can be installed on commodity hardware. Once installed, all maintenance can be
-            done using Rockstor's web-ui which doesn't expect the user to be a Linux Pro.
+            Rockstor, the first Linux BTRFS NAS software appliance, aims to combines flexibility & ease-of-use.
+            Generic commodity hardware, both <b>X86_64 and ARM64 (e.g. Pi4)</b>, can easily be up and running within minutes.
+            <b>Rockstor 4 is "Built on openSUSE"</b> while our near legacy 3 and older versions used CentOS 7 with newer elrepo kernels.
+            Once installed all maintenance is via Rockstor's Web-UI: good for NAS/linux beginners and pros alike.
```

Comment:
Minor style change for clarity:
```suggestion
            Once installed, all maintenance is done via Rockstor's Web-UI: good for NAS/linux beginners and pros alike.
```